pygmt/datatypes/dataset.py

```diff
@@ -153,6 +154,17 @@ def to_strings(self) -> np.ndarray[Any, np.dtype[np.str_]]:
             for segment in table.contents.segment[: table.contents.n_segments]:
                 if segment.contents.text:
                     textvector.extend(segment.contents.text[: segment.contents.n_rows])
+        if None in textvector:
+            # Workaround for upstream GMT bug reported in
+            # https://github.com/GenericMappingTools/pygmt/issues/3170.
+            msg = (
+                "The trailing text column contains `None' values and has been replace "
+                "with an empty string to avoid TypeError exceptions. "
```

Comment:
```suggestion
                "The trailing text column contains `None' values and has been replaced"
                "with empty strings to avoid TypeError exceptions. "
```